### PR TITLE
Issue #4: Add how to handle divergent branches

### LINKED ISSUE #4: Add Branch Reconciliation Guidelines to Contributing.md
In projects with multiple contributors, branches often diverge due to parallel development. Clear guidelines on how to reconcile these branches (using merge or rebase strategies) will ensure consistency and clarity in collaboration.

Without standardized procedures, contributors might use different methods to integrate changes, leading to unnecessary conflicts and complications. Providing a standard approach will help in avoiding such issues.

## Merge vs rebase
On shared branches, merging is often preferred. It avoids rewriting history, which can be problematic for others working on the same branch.

Rebasing before making a pull request can clean up your history, making it easier for maintainers to understand your changes.

I would recommend Merge since it does not change the existing history.

### PR DESCRIPTION
**Pull request by SonOfLope**
- [x] Adds guidelines on handling divergent branches, with a focus on using the rebase strategy for integrating changes.
- [x] Replaces screenshots not displaying.
- [x] Updates content to improve readibility.
- [x] Fixes formatting.

### CODE REVIEW

**Review by rngadam — APPROVED** [review]

**Comment by SonOfLope** [comment]
@rngadam is there a reason why this repo asks for 3 approving reviews ?
image

**Comment by rngadam** [comment]
@SonOfLope yes; we want a semi-consensus on rules and documentation that applies to the organization as a whole. 

BTW, ADRs should require approval from all active team members.

**Comment by SonOfLope** [comment]
> @SonOfLope yes; we want a semi-consensus on rules and documentation that applies to the organization as a whole.

Good to know. I will add reviewers. 

@JolanThomassin I saw you had to rebase for one of your pull request's, i'm adding you as a reviewer so you can give feedback if this is enough information to give directions for when it happens.

File: profile/CONTRIBUTING.md
Comment (JolanThomassin) [inline profile/CONTRIBUTING.md]: You need to rebase when the **main branch** receives new updates while you are **working on your branch**. This involves pulling the current state of the main branch into your pull request (PR) and **ensuring that everything works before merging your PR with the main branch**.

To be honest, I have primarily worked on solo projects, so I haven't encountered this situation frequently. Therefore, I'm not entirely certain if I'm the best person to provide advice. In my opinion, the text looks good. It might be helpful to use a graph to illustrate this process because it can greatly help in comprehension. Consider adding a graph and modifying the text slightly to explain why rebasing is recommended.

**Review by JolanThomassin — CHANGES_REQUESTED** [review]

File: profile/CONTRIBUTING.md
Comment (SonOfLope) [reply profile/CONTRIBUTING.md]: I appreciate the feedback! I'll try to cook something up.

**Comment by SonOfLope** [comment]
Changes were made to reflect suggestions!

**Review by rngadam — APPROVED** [review]

**Comment by rngadam** [comment]
@SonOfLope can you get your reviewers to look at it and get this merged ASAP

**Comment by SonOfLope** [comment]
@JolanThomassin whenever you have time, I added a step by step on how to proceed. 

Also @ThomasCardin whenever you have time.

**Comment by rngadam** [comment]
@SonOfLope Jolan will probably be unavailable for the next week so adding @ibrahim-kabir and @MaxenceGui

File: profile/CONTRIBUTING.md
Comment (MaxenceGui) [inline profile/CONTRIBUTING.md]: Here the branch name start with `issue59`, I think we agreed that now branch should start with only their number instead `59`.

File: profile/CONTRIBUTING.md
Comment (MaxenceGui) [inline profile/CONTRIBUTING.md]: Typo `ah` instead of `sh`

File: profile/CONTRIBUTING.md
Comment (MaxenceGui) [inline profile/CONTRIBUTING.md:244]: Explanation is clear and concise 👍🏻. Every steps to rebase in command line is well explain. Will there be a equivalent for interface?

For example:

git config --global pull.rebase true ; but you can also achieve with that specific `VScode GitHub extension feature`

**Review by MaxenceGui — CHANGES_REQUESTED** [review]

File: profile/CONTRIBUTING.md
Comment (elwoujdi) [reply profile/CONTRIBUTING.md:244]: It would be great to have the UI equivalent, save more time.

File: profile/CONTRIBUTING.md
Comment (JolanThomassin) [inline profile/CONTRIBUTING.md]: Why is there two part for "good issue"?

File: profile/CONTRIBUTING.md
Comment (JolanThomassin) [inline profile/CONTRIBUTING.md]: Screenshots don't works for me

**Review by JolanThomassin — CHANGES_REQUESTED** [review]

**Review by MaxenceGui — APPROVED** [review]

**Review by JolanThomassin — APPROVED** [review]

**Comment by rngadam** [comment]
@ibrahim-kabir you were asked a week ago to review

@MaxenceGui review and approve if there's not anything else.

**Pull request merged**
